src/test/util/test-rig.ts

```diff
@@ -318,7 +322,11 @@ class ExecResult {
       // own process group. Passing the negative of the child's pid kills all
       // processes in the group (without the negative only the leader "sh"
       // process would be killed).
-      process.kill(-this._child.pid, 'SIGINT');
+      if (force) {
+        process.kill(-this._child.pid, 9);
```

Comment:
Nit: thoughts about using `'SIGKILL'` instead of the number?

Reply:
Yeah that seems better